src/Installer.php

```diff
@@ -544,6 +632,10 @@ private function parseOptions(array $argv)
         $this->version = false;
         $this->filename = 'puli.phar';
         $this->cafile = false;
+        $this->stability = 'unstable';
+        if (in_array('--prefer-stable', $argv)) {
```

Comment:
Let's change this to `--stable`. If this option is passed, we don't actually "prefer" a stable version, but we enforce loading of a stable version (so it's rather like `--stability=stable`, but shorter).